pages/examples/target-is-visible-exuqdg.md

```diff
@@ -0,0 +1,183 @@
+---
+id: exuqdg
+name: Target is visible
+rule_type: atomic
```

Comment:
All of this makes this look like a regular rule. I wonder if we should do some things to this that move it away from that; in part not to confuse people, and in part just because it isn't that relevant. Like, I'm not sure this needs accessibility requirements, input aspect, applicability, expectation, etc. 

```suggestion
rule_type: definition-test
```